ocaml/idl/datamodel.ml

```diff
@@ -3902,8 +3902,11 @@ let task =
 
 (** Many of the objects need to record IO bandwidth *)
 let iobandwidth =
-  [ field ~persist:false ~qualifier:DynamicRO ~ty:Float "read_kbs" "Read bandwidth (KiB/s)";
-    field ~persist:false ~qualifier:DynamicRO ~ty:Float "write_kbs" "Write bandwidth (KiB/s)" ]
+  [ field ~persist:false ~qualifier:DynamicRO ~ty:Float "read_kbs" "Read bandwidth (KiB/s)"
+      ~lifecycle:[ Removed, rel_ely, "Disabled in favour of RRDs"]
+  ; field ~persist:false ~qualifier:DynamicRO ~ty:Float "write_kbs" "Write bandwidth (KiB/s)"
+      ~lifecycle:[ Removed, rel_ely, "Disabled in favour of RRDs"]
+  ]
 
```

Comment:
As I understand it, the field is not really removed. I was following the example set by `vm_guest_metrics.memory`.